docs/rst/amldashboard/user_manual/usage.rst

```diff
@@ -0,0 +1,237 @@
+
+.. figure:: /rst/figures/amldashboard/data_management.png
+
+Training Tab
+^^^^^^^^^^^^
+
+To train a model using |aml|, follow these steps:
+
+.. note::
+   Please ensure that at least one **Computing Node** is running to facilitate the training process.
+
+1. Specify the number of parallel trainings (executions) you wish to run.
+
+2. Define the number of iterations per execution.
+
+3. Set the percentage of dataset to distribute in each execution.
+
+4. Click on the ``Train`` button in the *AML Training Launcher* to initiate the training process.
+
+5. Once the training is completed, the model status will appear as **Finished :)** in the *AML Status*.
+
+6. If the training process fails, an error message will be displayed indicating the reason for the failure.
+
+
+.. figure:: /rst/figures/amldashboard/training.png
+
+Fetching Tab
+^^^^^^^^^^^^
+
+To fetch a model, follow these steps:
+
+.. note::
+
+   Make sure that at least one **Model Manager Sender Node** is running to facilitate the model fetching process.
+
+1. Click on the ``Search for statistics`` button in the *AML Statistics Fetcher*.
+
+2. Once the statistics are received, the status will appear as **Statistics received !** in the *AML Collaborative Learning Status*.
+
+3. Click on the ``Request model`` button in the *AML Model Fetcher*.
+
+4. Once the model is received, the status will change to **Model received !** in the *AML Collaborative Learning Status*.
+
+5. If the fetching process fails, an error message will be displayed indicating an error.
+
+
+.. figure:: /rst/figures/amldashboard/fetching.png
+
+Batch Prediction Tab
+^^^^^^^^^^^^^^^^^^^^
+
+To predict the output of a dataset, follow these steps:
+
+.. note::
+
+   Ensure that you have access to at least one |aml| Model to facilitate the batch prediction process.
+
+1. Click on the ``Update predictions`` button in the *Algebraic Machine Learning*.
+
+2. The predictions will be displayed in the Results *Algebraic Machine Learning* plot.
+
+.. figure:: /rst/figures/amldashboard/batch_prediction.png
+
+Real-Time Prediction Tab
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To predict the output of webcam images in real^time using an |aml| Model, follow these steps:
+
+.. note::
+
+   Ensure that you have access to at least one |aml| Model to facilitate the real^time prediction process.
+
+1. Toggle the ``prediction`` switch in the *Predict for AML* section and activate the ``video`` switch in the webcam section.
+
+2. The predictions will be displayed in the *Results AML* plot.
+
+
+.. figure:: /rst/figures/amldashboard/real_time_prediction.png
+
+Context Broker Tab
+^^^^^^^^^^^^^^^^^^
+
+To create and update data to the Context Broker and get the solution (inference) from the |aml| Model, follow these steps:
+
+.. note::
+
+   Make sure that the Context Broker is running to ensure successful interaction and data exchange.
```

Comment:
Add a reference to the context broker installation guide to facillitate the use.

Reply:
Done